configuration.py

```diff
@@ -6,7 +6,7 @@ class Config:
     ec2_region = ""
     ec2_amis = ['']
     ec2_keypair = ''
-    ec2_secgroups = ['']
+    ec2_secgroups = [''] #ID Not Name
```

Comment:
Security groups names can be used in the case of an EC2 security group, I imagine the ID must be specified for IAM roles (explaining your error). Make this difference clear.